AutomatedTesting/Gem/PythonTests/Atom/atom_utils/atom_tools_utils.py

```diff
@@ -138,7 +145,8 @@ def get_last_model_preset_path() -> str:
     "C:/git/o3de/Gems/Atom/Tools/MaterialEditor/Assets/MaterialEditor/ViewportModels/Cone.modelpreset.azasset"
     "C:/git/o3de/Gems/Atom/Tools/MaterialEditor/Assets/MaterialEditor/ViewportModels/BeveledCone.modelpreset.azasset"
     """
-    return azlmbr.atomtools.EntityPreviewViewportSettingsRequestBus(azlmbr.bus.Broadcast, "GetLastModelPresetPathWithoutAlias")
+    return azlmbr.atomtools.EntityPreviewViewportSettingsRequestBus(
+        azlmbr.bus.Broadcast, "GetLastModelPresetPathWithoutAlias")
```

Comment:
Backlog: We should probably get rid of writing strings out for these bus calls and start building classes that store these strings based upon their ownership group.

If that bus changes, the whack-a-mole for changing the string value passed could cause a huge headache for maintenance down the line.

Reply:
Thanks for the feedback that's a valid point. Cut a GHI that I will get to next sprint: https://github.com/o3de/o3de/issues/13544